CONTRIBUTING.md

```diff
@@ -8,12 +8,13 @@ defaang lives through contributions from the community. When contributing to thi
   - [Where to Get Help](#where-to-get-help)
     - [All Communication Channels](#all-communication-channels)
   - [How to Help](#how-to-help)
-    - [Our Stack (with linked documentation)](#our-stack)
+    - [Our Stack](#our-stack)
+    - [Project Board](#project-board)
     - [Issue Labels](#issue-labels)
-    - [Semantic Commits](#semantic-commits)
-    - [Commit Example](#commit-example)
-  - [Making Changes](#making-changes)
-  - [Pushing Changes](#pushing-changes)
+    - [Semantic commits](#semantic-commits)
```

Comment:
ohh... this was generated automatically by the table of contents itself because [line: 66](https://github.com/benny123tw/defaang/blob/docs/CONTRIBUTING/CONTRIBUTING.md?plain=1#L66) `commits` is lowercase

Reply:
updated: [6fe641e](https://github.com/ykdojo/defaang/pull/119/commits/6fe641e8d98ed1424823d302c993110d7848796b)